.github/workflows/ci-for-diff-ns.yaml

```diff
@@ -56,6 +56,18 @@ jobs:
           kubectl-rook_ceph -o test-operator -n test-cluster rook purge-osd 0 --force
           kubectl rook_ceph -o test-operator -n test-cluster health
 
+          # for testing start-debug and stop-debug deployment
```

Comment:
How about checking that the deployments are created, the debug pods are running, and the original pods are stopped?

Reply:
Working on this https://github.com/rook/kubectl-rook-ceph/issues/42

Actually in the whole CI there are random sleep so need to update that too